### ISSUE TITLE
Flakeheaven is incompatible with mkdocs due to importlib-metadata

### ISSUE THREAD

**Issue by billsioros** [flakeheaven/flakeheaven#89]
While running `poetry install` I face the following issue:

```sh
Because no versions of mkdocs match >1.3.0,<2.0.0
   and mkdocs (1.3.0) depends on importlib-metadata (>=4.3), mkdocs (>=1.3.0,<2.0.0) requires importlib-metadata (>=4.3).
  And because flakeheaven (0.11.1) depends on importlib-metadata (>=1.0,<2.0)
   and no versions of flakeheaven match >0.11.1,<0.12.0, mkdocs (>=1.3.0,<2.0.0) is incompatible with flakeheaven (>=0.11.1,<0.12.0).
  So, because cookiecutter-pypackage depends on both flakeheaven (^0.11.1) and mkdocs (^1.3.0), version solving failed.
```

I tried downgrading both packages to different versions but failed to find a compatible combination.

Thanks in advance!

**Comment by pwoolvett** [comment]
related https://github.com/flakeheaven/flakeheaven/issues/80

fixed by #82 

you can either use main branch  for the moment, or wait for next release

**Comment by billsioros** [comment]
ty for the immediate reply! and sorry for not noticing the duplicate!
